__init__.py

```diff
@@ -66,13 +66,13 @@ def get_bin_directory():
     raise Exception(msg + "\n\nImportError: %s" % error) from None
 
 bl_info = {
-    "name": "LuxCoreRender is a physically based and unbiased rendering engine",
-    "author": "Odilkhan Yakubov (odil24), Simon Wendsche (B.Y.O.B.), Michael Klemm (neo2068), acasta69, u3dreal, Philstix",
-    "version": (2, 61.1),
-    "blender": (3, 0, 0),
+    "name": "LuxCoreRender",
+    "author": "Simon Wendsche (B.Y.O.B.), Michael Klemm (neo2068), Odilkhan Yakubov (odil24), acasta69, u3dreal, Philstix",
+    "version": (2, 61),
+    "blender": (3, 4, 0),
```

Comment:
Are you removing support for Blender 3.0, 3.1, 3.2, and 3.3?

Reply:
Thats because LuxCore (latest) works flawless from Blender 3.4 than previous versions which is not supported new "index_attribute" operator which fixed by @neo2068 for material indexing (Multi-material fixed):
https://github.com/LuxCoreRender/BlendLuxCore/commit/6318b6caa906946eda0852dcf420b88f52810d97

Reply:
> Are you removing support for Blender ~3.0~, 3.1, 3.2, and 3.3?
> 
> We usually like to retain support at least for the LTS version (3.3), and even older versions if there is no technical reason why they should be incompatible.

Sorry, but I don't understand what here a link is written because I'm not familiar with that.

Reply:
> Are you removing support for Blender ~3.0~, 3.1, 3.2, and 3.3?

No, the versions are still supported. I changed the version back to 3.0. I had interpreted the version as the latest supported version but in the documentation it is listed as minimum supported version.

Reply:
OK that's great, thanks @neo2068

Reply:
@neo2068 hi. Why are you did not include my latest changes related to the icons? Is my idea bad?